k8s/k8s-aws-efs-csi-driver/iam.tf

```diff
@@ -0,0 +1,73 @@
+
+locals {
+  eks_oidc_short = replace(var.kubernetes_cluster.data.infrastructure.oidc_issuer_url, "https://", "")
+  iam_resources  = length(var.storage_class_to_efs_arn_map) > 0 ? [for sc_name, efs_arn in var.storage_class_to_efs_arn_map : efs_arn] : ["*"]
+}
+
+data "aws_arn" "eks_cluster" {
+  arn = var.kubernetes_cluster.data.infrastructure.arn
+}
+
+resource "aws_iam_role" "efs-csi-controller" {
```

Comment:
Can we name the resources consistently w underscores or hyphens?

Reply:
Done